bundles.config.js

```diff
@@ -765,6 +765,15 @@ exports.extensionBundles = [
     options: {hasCss: true},
     type: TYPES.MISC,
   },
+  {
+    name: 'amp-shaka',
+    version: '0.1',
+    latestVersion: '0.1',
+    type: TYPES.MEDIA,
+    postPrepend: [
+      'third_party/shaka-player/shaka-player.compiled.js',
```

Comment:
We cannot run 3rd party code on the top window. This should be loaded on an nested `3p` frame, [see `amp-facebook` for an example.](https://github.com/ampproject/amphtml/blob/a8c274dce52d9550e1f9d57d273a5c5e8fab7258/extensions/amp-facebook/0.1/amp-facebook.js#L75)